src/main/java/org/takes/rq/RqMultipart.java

```diff
@@ -63,6 +64,15 @@
 public interface RqMultipart extends Request {
 
     /**
+     * Carriage return constant.
+     */
+    String CRLF = "\r\n";
+    /**
+     * Content type constant.
+     */
+    String CONTENT_TYPE = "Content-Type";
```

Comment:
@erimerturk let's suppress Qulice than, but exposing internal constants in a public interface is a bad idea. Our users will be confused by a public constant `RqMultipart.CONTENT_TYPE`.. what is it about, they will ask?